CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# dbt_unified_rag v0.1.0-a4
+
+## Breaking Changes
+- Added the hubspot `engagement` source table to the package and made the following updates:
+    - Added `stg_rag_hubspot__engagement` model as part of the hubspot staging models.
+    - Updated `int_rag_hubspot__deal_document` to adjust the method that `hubspot_engagement_*` models are joined by leveraging the `hubspot__engagement` table as the intermediary joining table for the `engagement_contact` and `engagement_company` tables.
```

Comment:
```suggestion
    - Updated `int_rag_hubspot__deal_document` joins so that `stg_rag_hubspot__engagement` table served as the intermediary joining table for the `stg_rag_hubspot__engagement_contact` and `stg_rag_hubspot__engagement_company` tables.
```

This was a bit confusing to read. I reworded it, let me know your thoughts.